ts/modules/accessibility/focusBorder.ts

```diff
@@ -73,9 +73,27 @@ extend(H.SVGElement.prototype, {
         bb.x += this.translateX ? this.translateX : 0;
         bb.y += this.translateY ? this.translateY : 0;
 
+        let borderPosX: number,
+            borderPosY: number;
+
+        // For text elements, apply x and y offset, #11397.
+        if (this.element.nodeName === 'text') {
+            const isRotated = Boolean(this.rotation);
```

Comment:
`!!this.rotation` will give you the same output and is more common in the rest of the code.